info/v1/container.go

```diff
@@ -899,6 +899,9 @@ type ContainerStats struct {
 
 	// Statistics originating from perf events
 	PerfStats []PerfStat `json:"perf_stats,omitempty"`
+
+	// Working set size
+	Wss uint64 `json:"wss,omitempty"`
```

Comment:
Should we nest this under MemoryStats?

Reply:
All memory stats come from memory cgroup so I think that wss does not fit to them.